README.md

```diff
@@ -16,6 +16,9 @@ from source with `cargo doc`. Examples are included in the docs.
 
 ## Changes
 
+### `0.5.0`
```

Comment:
Since the latest version in crates.io is `0.3.0`, these changes can be added to the pending `0.4.0` release, which I will publish as soon as possible.

Reply:
My bad, didn't notice the discrepancy between the Github and crates.io versions. I'll downgrade back to 0.4.0.